src/stratis_cli/_actions/_data.py

```diff
@@ -220,29 +221,43 @@ def interface_name_to_common_name(interface_name):
     raise StratisCliValueError(interface_name, "interface_name")
 
 
+# Accepted STRATIS_DBUS_TIMEOUT environment variable values are:
+# 1. an integer between 0 (inclusive) and INT_MAX (inclusive),
+#    which represents the timeout length in milliseconds
+# 2. any negative integer, which represents a sane default that was
+#    supplied by libdbus (handled by dbus-python)
+
 try:
+
+    timeout = int(environ.get("STRATIS_DBUS_TIMEOUT", DBUS_TIMEOUT_SECONDS * 1000))
+
+    # Value too large
+    if timeout > (1 << 31) - 1:
+        raise ValueError("The timeout value too large.")
```

Comment:
You will have to define a new StratisCli exception. I think it might be best to call it ```StratisCliEnvironmentError.``` You can define it just like ```StratisCliGenerationError``` is defined, at least for now, and raise it in the same way as ```StratisCliGenerationError``` is raised in this file.

You will then have to figure out how to catch it properly within the source.